hazelcast/include/hazelcast/client/internal/nearcache/impl/store/NearCacheObjectRecordStore.h

```diff
@@ -46,27 +46,25 @@ class NearCacheObjectRecordStore
       HeapNearCacheRecordMap<K, V, KS, record::NearCacheObjectRecord<V>>>
       ANCRS;
 
-    NearCacheObjectRecordStore(const std::string& name,
```

Comment:
Could we revert this change? As it is public method, we should not change the signature. We might use another way to suppress the warning.